addons/web/static/src/js/services/crash_manager.js

```diff
@@ -121,34 +204,41 @@ var CrashManager = core.Class.extend({
                                     })
                             });
             }
-
-            this.show_warning(error);
-        //InternalError
-
+            this.show(error);
         } else {
             this.show_error(error);
         }
     },
-    show_warning: function(error) {
+    /**
+     * overrid in website to display a toast.
```

Comment:
We never write that kind of comment, we don't mention apps which are not a dependency of the current app. And why not overridding `show_warning` then instead of creating a new method which wraps it ?

Reply:
I did not override ```show_warning``` because I taught it could be useful.
But now that i rethink about it, it is not and it should be overrode